### PR TITLE
[CI] Show CI author partial/full name depending on user [ch819]

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: tests/Test/Geocoder/DummyGeocoder.php [hunk 1]
```diff
@@ -61,6 +61,10 @@ class DummyGeocoder implements GeocoderInterface
             'lat' => 48.866667,
             'lon' => 2.333333,
         ],
+        'paris 9e, france' => [
+            'lat' => 48.8790183,
+            'lon' => 2.3379062,
+        ],
```
Comment (Remg) [inline tests/Test/Geocoder/DummyGeocoder.php]: is it related to this PR ?

File: tests/Controller/EnMarche/CitizenInitiativeControllerTest.php [hunk 2]
```diff
@@ -38,6 +38,34 @@ public function testAnonymousUserCannotCreateCitizenInitiative()
...
+        // Anonymous
+        $crawler = $this->client->request(Request::METHOD_GET, '/evenements');
+
+        $this->assertSame(2, $crawler->filter('.search__results__meta span:contains("Jacques P.")')->count());
+        $this->assertSame(0, $crawler->filter('.search__results__meta span:contains("Jacques Picard")')->count());
+
+        $crawler = $this->client->request(Request::METHOD_GET, '/initiative_citoyenne/'.LoadCitizenInitiativeData::CITIZEN_INITIATIVE_3_UUID.'/'.date('Y-m-d', strtotime('tomorrow')).'-apprenez-a-sauver-des-vies');
+
+        $this->assertSame('Organisé par Jacques P.', trim(preg_replace('/\s+/', ' ', $crawler->filter('.committee-event-organizer')->text())));
+    }
```
Comment (Remg) [inline tests/Controller/EnMarche/CitizenInitiativeControllerTest.php]: You could ensure the full name never appears on the page by crawling all the content and asserting "Jacques Picard" does not appear at all, not only in the search results.

File: tests/Test/Geocoder/DummyGeocoder.php (hunk 1 above)
Comment (illusionOfParadise) [reply tests/Test/Geocoder/DummyGeocoder.php]: I need this for new tests.

File: tests/Controller/EnMarche/CitizenInitiativeControllerTest.php (hunk 2 above)
Comment (illusionOfParadise) [reply tests/Controller/EnMarche/CitizenInitiativeControllerTest.php]: The full name can appear for events.

Comment (Remg) [reply tests/Controller/EnMarche/CitizenInitiativeControllerTest.php]: fair enough !

**Pull request merged**